var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -41,6 +41,9 @@ class Ncurses(AutotoolsPackage):
     variant('symlinks', default=False,
             description='Enables symlinks. Needed on AFS filesystem.')
 
+    variant('wide', default=True,
+            description='Enable to build ncuresesw, disable to build ncurses')
```

Comment:
Can we build both at the same time?

We do this e.g. for `fftw`, which has variants for single and double precision. By building both, packages can choose what they like best. If `ncurses` and `ncursesw` use different library names, then they don't actually conflict, do they?